lib/user/mbaas-service-proxy.js

```diff
@@ -0,0 +1,126 @@
+var q = require('q')
+  , $fh = require('fh-mbaas-api')
+  , _ = require('lodash');
+
+/**
+ * Proxy for calling MBaaS Service API
+ * establishing an independent connection channel between
+ * Cloud App and MBaaS Service.
+ * @param guid name of MBaas Service
+ * @constructor
+ */
+var MbaasServiceProxy = function(guid) {
+  this.guid = guid;
+};
+
+/**
+ * MBaaS service API call
+ * @param _options we want to overwrite
+ * use default options otherwise
+ *  - guid, name of mbaas service
+ *  - path, URL of MBaaS Service API
+ *  - method, HTTP method {GET, PUT}
+ * @returns {*|promise}
+ */
+MbaasServiceProxy.prototype.xhr = function(_options) {
+  var defaultOptions = {
+    guid: this.guid,
+    path: '/api/wfm/user',
+    method: 'GET'
+  };
+  var options = _.defaults(_options, defaultOptions);
+  var deferred = q.defer();
+  $fh.service(options, function(err, data) {
+    if (err) {
+      deferred.reject(err);
+      return;
+    }
+    deferred.resolve(data);
+  });
+  return deferred.promise;
+};
+
+MbaasServiceProxy.prototype.list = function() {
```

Comment:
I was thinking if add comments to any of them or just xhr() one cause we only pass different options to xhr